app/templates/signin.hbs

```diff
@@ -1,22 +1,24 @@
-<form>
-    <h1 class="formHeader"> Sign in </h1><br>
-    <div class="loginGroup">
-        <input type="email" name="useremail" required><span class="highlight"></span><span class="bar"></span>
-        <label>User Email</label>
-    </div>
-    <div class="loginGroup">
-        <input type="password" name="psw" required><span class="highlight"></span><span class="bar"></span>
-        <label>Password</label>
-        <div class="notification">
-            <i class="fa fa-exclamation-circle" aria-hidden="true"></i>The initial password should be the sign-in code sent to your email by the Block Project team.
-            <a href="#">
-                <span class="getCode"> Didn't get a code? </span>
-            </a>
+{{#signin-code}}
```

Comment:
This is a crappy diff, but the only change I made here was to add `{{#signin-code}}` to the top of the file, and `{{/signin-code}}` to the bottom of the file, and indent everything in the middle.

You can think of it just like an HTML element (`{{#signin-code}}{{/signin-code}}` instead of `<div></div>`), except it comes with code that can run also.